eth_account/ledger.py

```diff
@@ -0,0 +1,124 @@
+import struct
+
+import ledgerblue.comm
+
+ETH_DERIVATION_PATH_PREFIX = "44'/60'/0'/"
+
+class LedgerAccount:
+    """
+    Ledger Ethereum App Protocol Spec is located at:
+    <https://github.com/LedgerHQ/blue-app-eth/blob/master/doc/ethapp.asc>
+
+    References:
+    - https://github.com/LedgerHQ/blue-app-eth/blob/master/getPublicKey.py
+    - https://github.com/bargst/pyethoff/blob/master/tx_sign.py
+
+    """
+
+    def __init__(self, offset=None, address='')
+        self.dongle = ledgerblue.comm.getDongle(debug=False)
+
+        if offset is None and :
+            # Convert an address to an offset
+            if address == '':
+                raise Exception('Invalid offset and address provided')
+            else:
+                #TODO check address validity
+                self.offset = self.get_offset(address)
+                self.address = address
+        else:
+            self.offset = offset
+            self.address = self.get_addresses(offset)
+
+
+    def _parse_bip32_path(self, offset):
+        """
+        Convert an offset to a bytes payload to be sent to the ledger 
+        representing bip32 derivation path.
+        """
+        path = ETH_DERIVATION_PATH_PREFIX + str(offset)
+        result = bytes()
+        elements = path.split('/')
+        for pathElement in elements:
+            element = pathElement.split("'")
+            if len(element) == 1:
+                result = result + struct.pack(">I", int(element[0]))            
+            else:
+                result = result + struct.pack(">I", 0x80000000 | int(element[0]))
+        return result
+
+    def get_addresses(self, offset):
```

Comment:
The name of this function suggests that it will return multiple addresses, but it seems to only return a single one.